drivers/src/drv8825.cpp

```diff
@@ -1,16 +1,18 @@
-#pragma once
-#include <drv8825.hpp>
+#include "../include/drv8825.hpp"
```

Comment:
```suggestion
#include <drv8825.hpp>
```

Does this not work? If so, may be a good idea to enable this.